x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_details/monitor_errors/errors_icon.tsx

```diff
@@ -11,5 +11,5 @@ import { useMonitorErrors } from '../hooks/use_monitor_errors';
 export const MonitorErrorsIcon = () => {
   const { hasActiveError } = useMonitorErrors();
 
-  return hasActiveError ? <EuiIcon type="alert" color="danger" /> : null;
+  return hasActiveError ? <EuiIcon type="error" color="danger" /> : null;
```

Comment:
Hey @justinkambic! Its a great question. Intent is important here and its ultimately up to your team to decide which icon makes the most sense in your application's flow. The (yet to be documented) guidance from a design system perspective is as follows:

- The `error` icon should be used to indicate blocking issues that prevent a user from moving forward in a workflow
- The `warning` icon should be used to caution users against something that _should_, but does not necessarily _need_, to be addressed before moving forward.

I hope that helps clarify our direction. Our suggestion in the lines above were inferred from the surrounding code. We're happy to change this implementation if `warning` makes more sense in your case.

Reply:
https://github.com/elastic/kibana/pull/152506/commits/9b2eb8f4c37e32063ce637c1382ee141ae44e2c2 I've switched this back to the `warning` (old alert) icon per your comment @justinkambic, but just want to reiterate what Davey said - the renames aren't just renames, they're also requests for teams and designers to reconsider your usage across the board and whether you're using the right icon for the right scenario. It might be worth revisiting (after this PR merges) with your designer.

Reply:
This isn't actually an error in workflow, but an indication of degradation in the synthetic monitor itself and meant to warn users that they may need to review errors in the services they are monitoring.

Reply:
Gotcha, thanks for taking the time to confirm Dominique!

Reply:
++ thanks for the added context @dominiqueclarke ❤️